lib/ddlog_bigint.rs

```diff
@@ -20,6 +16,14 @@ use std::ops::*;
 use std::os::raw::c_char;
 use std::str::FromStr;
 
+/* This module is designed to be imported both as a standard DDlog library and as a normal Rust
+ * module, e.g., from `differential_datalog_test`.  We therefore need to import these traits
+ * so that they are available in the latter case and rename them so that they don't cause duplicate
+ * import error in the former case. */
+use differential_datalog::record::FromRecord as FromRec;
+use differential_datalog::record::IntoRecord as IntoRec;
+use differential_datalog::record::Record;
+
 #[derive(Eq, PartialOrd, PartialEq, Ord, Clone, Hash)]
 pub struct Int {
```

Comment:
Implementing `Deref<Target = BigInt>` would add to ergonomics a lot (Same with `Uint`)

Reply:
As long as it doesn't mess up the generated code. There's a bit of a tension here between ergonomics and surprising side effects in automatically generated code where fewer magic conversions is better. The same is true for many other library types. I don't think we have a consistent approach here. I'll try implementing these `Deref` and see if anything breaks.

Reply:
If it ends up that there's a multitude of unwanted behavior happening, we can delay that and I can make a pr that makes you ye generated code use more qualified function calls, but hopefully it doesn't come to that